### PR TITLE
Make new preset format human-readable

### PR DESCRIPTION
**Pull request by dimitry-**
And enable auto-conversion from old to new format on successful preset load.

Bug: http://b/168799822
Test: Load preset in old format, check that it is in new format
      and human readable.

### CODE REVIEW

**Comment by dimitry-** [comment]
first commit is duplicate of #2316 you can look only at the second commit.

File: src/OrbitGl/PresetFile.cpp [hunk 1]
```diff
@@ -4,12 +4,46 @@
...
 
+#include <absl/strings/match.h>
+#include <google/protobuf/io/zero_copy_stream_impl_lite.h>
+#include <google/protobuf/text_format.h>
+
+#include "OrbitBase/File.h"
 #include "OrbitBase/Logging.h"
+#include "OrbitBase/ReadFileToString.h"
 
 namespace orbit_gl {
```
Comment (dpallotti) [inline src/OrbitGl/PresetFile.cpp:15]: Not related to this change, but what's the reason for this to be in `OrbitGl`?

File: src/OrbitGl/App.cpp [hunk 2]
```diff
@@ -1773,16 +1745,15 @@ void OrbitApp::LoadPreset(const PresetFile& preset_file) {
...
-      //            absl::StrFormat("Unable to convert preset file \"%s\" to new file format: %s",
-      //                            preset_file.file_path().string(),
-      //                            convertion_result.error().message()));
-      //      }
+    } else {
+      // Then if load was successful and the preset is in old format - convert it to new one.
+      auto convertion_result = ConvertPresetToNewFormatIfNecessary(preset_file);
+      if (convertion_result.has_error()) {
+        SendWarningToUi(
+            "Conversion to new format failed",
```
Comment (dpallotti) [inline src/OrbitGl/App.cpp:1753]: Should we really show this to the user?

**Review by dpallotti — APPROVED** [review]

File: src/OrbitGl/PresetFile.cpp (hunk 1 above)
Comment (dimitry-) [reply src/OrbitGl/PresetFile.cpp:15]: I guess it wants to be its own component... I will create a followup CL for this

File: src/OrbitGl/App.cpp (hunk 2 above)
Comment (dimitry-) [reply src/OrbitGl/App.cpp]: Converted to ERROR, also added the code to restore backed up file in case SaveToFile fails.

Comment (dimitry-) [reply src/OrbitGl/App.cpp]: not sure where my comment went. I converted it to ERROR and added "restore from .backup" functionality to ConvertPresetToNewFormatIfNeccessary().

**Pull request merged**